packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -65,7 +65,7 @@ const MaxReconnectDelaySeconds = 8;
 const InitialReconnectDelaySeconds = 1;
 const MissingFetchDelaySeconds = 0.1;
 const MaxFetchDelaySeconds = 10;
-const MaxBatchDeltas = 2000;
+const MaxBatchDeltas = 5000; // Please see Issue #5211 for data around batch sizing
```

Comment:
Should we make this driver configurable?  I assume that r11s/frs might have different perf characteristics.
 @tanviraumi

Reply:
My plan to innovate here for another version (I'll add multiple requests in parallel), but then move it all into driver.
I believe driver needs to expose streaming API, there should be no chunks at all here. I.e. DM should be able to ask for 1M ops and receive them in whatever chunks driver decided to do.

Reply:
If I remember correctly, the reason we did 2000 was to get around "Axios - MaxBodyLength" error. I agree on the future direction that it should be up to driver to decide how many to fetch and deltaManager should just rely on a streaming api. In the meantime, can this number be per driver?

Reply:
Instead of making it per-driver, I'd rather make any driver that needs to have smaller policy just use smaller batch.
I.e. r11s could  (instead of fetching 5K ops) fetch 2K ops and set partialResult = (length === 2000) on response to indicate that it did not fulfil fully request.

That's one step on streaming direction - client askes infinite, server returns smaller chunk

I can make this change, is this only r11s that has that limit?
And also - is this limit because we are not using compression? Cause "normal" 2K ops is 80K compressed on average :) And even on upper end (all ops are chunked ops), it's around 2Mb in size compressed.

Reply:
Sounds good! Feel free to make the change.

I am not sure about ODSP. But yes, We don't do compression today, something we are planning to tackle soon. I don't remember whether it was Axios or nginx having that limit. Will need to check back on that

Reply:
Done - take a look.
You definitely want to enable compression here, and most likely - on most other APIs. There is a huge difference for someone on slow network, and event taking into account decompression cost (on slow machine) it's almost always net benefit.